platform.md

```diff
@@ -276,10 +275,58 @@ All lifecycle phases:
 - MUST read `CNB_PLATFORM_API` from the execution environment and evaluate compatibility before attempting to parse other inputs (see [Platform API Compatibility](#platform-api-compatibility))
 - MUST give command line inputs precedence over other inputs
 
+#### `analyzer`
+Usage:
+```
+/cnb/lifecycle/analyzer \
+  [-analyzed <analyzed>] \
+  [-daemon] \ # sets <daemon>
+  [-gid <gid>] \
+  [-group <group>] \
```

Comment:
From the lifecycle PR: it looks like group is only defined for earlier platform apis, is that intended?

Reply:
Now that I'm remembering how all of this works... we can't have group if analyze goes first...
```suggestion
```

Reply:
We can have `group`, but not `order`

Reply:
I think the future we will support `stack-group` in order to validate mixins, but not `group`?

Reply:
I think I confused myself with the names. We need `<order>`, not `<group`>. Fixed in 6480d2e975ba81d5b61d681f0177539408c834b8